plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/catalog/TrinoCatalog.java

```diff
@@ -196,4 +197,9 @@ void createMaterializedView(
     void updateColumnComment(ConnectorSession session, SchemaTableName schemaTableName, ColumnIdentity columnIdentity, Optional<String> comment);
 
     Optional<CatalogSchemaTableName> redirectTable(ConnectorSession session, SchemaTableName tableName, String hiveCatalogName);
+
+    default Set<String> getUpdatableTableProperties()
```

Comment:
Is this an extension point for your fork?

Reply:
It was a simple change that could be contributed back to OSS hence opened up for here.

Reply:
@ebyhr and then I'll backport this change if this goes through. 
but even though I am a member of the trino community and I did the CLA thing, not sure why it says i am not authorized 🤔

Reply:
First of all, I am not convinced with this change. We usually deny fork-specific changes. 

What do you mean > "i am not authorized"?